pkg/apis/application/v1alpha1/applicationset_types.go

```diff
@@ -86,6 +86,7 @@ type ApplicationSetStrategy struct {
 	Type        string                         `json:"type,omitempty" protobuf:"bytes,1,opt,name=type"`
 	RollingSync *ApplicationSetRolloutStrategy `json:"rollingSync,omitempty" protobuf:"bytes,2,opt,name=rollingSync"`
 	// RollingUpdate *ApplicationSetRolloutStrategy `json:"rollingUpdate,omitempty" protobuf:"bytes,3,opt,name=rollingUpdate"`
+	DeletionOrder string `json:"deletionOrder,omitempty" protobuf:"bytes,3,opt,name=deletionOrder"`
```

Comment:
Mind incrementing 3 to 4 for the protobuf field number.

Reply:
protobuf field number should be 3, line 88 is commented. 
I can remove the previously commented field or move the `DeletionOrder` field above the comment to avoid confusion for anyone reading the code.

Reply:
Oh, I missed that, you are right. Disregard